Wei/View/MyWallet/LatestTransactionList/LatestTransactionListViewModel.swift

```diff
@@ -64,19 +66,59 @@ final class LatestTransactionListViewModel: InjectableViewModel {
         
         let getTransactionsAction = Driver.merge(getTransactionAction, refreshTransactionAction)
         
-        let (latestTransactions, error) = (
-            getTransactionsAction.elements.map { response -> [TransactionHistory] in
-                return response.elements
-                    // filters transactions executed more than a day ago,
-                    // and shows only the first 5th.
-                    .filter { $0.isExecutedLessThanDay }
-                    .reversed()
-                    .prefix(5)
-                    .map { TransactionHistory(transaction: $0, myAddress: myAddress) }
-            },
+        // Local transactions saved in realm database, a local transaction will be deleted
+        // when transactions get included into a block
+        let updateTick = localTransactionRepository.updateTick.asDriver(onErrorDriveWith: .empty())
+        let localTransactions = updateTick.startWith(())
+            .flatMap { [weak self] _ -> Driver<[LocalTransaction]> in
+                guard let weakSelf = self else {
+                    return Driver.empty()
+                }
+                let localObjects = weakSelf.localTransactionRepository.objects()
+                    .sorted { $0.date > $1.date }
+                
+                return Driver.just(localObjects)
+            }
+            .do(onNext: { [weak self] localTransactions in
+                // Delete local transactions which are older than 1 hour.
+                localTransactions
+                    .filter { Date(timeIntervalSince1970: TimeInterval($0.date)).timeIntervalSinceNow < Double(-3600.0) }
+                    .forEach { [weak self] localTransaction in
+                        self?.localTransactionRepository.delete(primaryKey: localTransaction.txID)
+                    }
+            })
+        
+        let (transactions, error) = (
+            getTransactionsAction.elements.do(onNext: { [weak self] transactions in
+                transactions.elements.forEach {
+                    // delete local transaction if the same txID is included in one of the transactions
+                    self?.localTransactionRepository.delete(primaryKey: $0.hash)
+                }
+            }),
             getTransactionsAction.error
         )
         
+        // Create TransactionHistoryKind model from local transactions and remote transactions
+        let transactionHistoryKinds = Driver
+            .combineLatest(
+                localTransactions.map { $0
+                    .map { TransactionHistoryKind.local($0) }
+                },
+                transactions.map { $0.elements
+                    .filter { $0.isExecutedLessThanDay }
```

Comment:
I think should be comment `// filters transactions executed more than a day ago`